src/ng/q.js

```diff
@@ -381,7 +381,11 @@ function qFactory(nextTick, exceptionHandler, errorOnUnhandledRejections) {
       if (!toCheck.pur) {
         toCheck.pur = true;
         var errorMessage = 'Possibly unhandled rejection: ' + toDebugString(toCheck.value);
-        exceptionHandler(errorMessage);
+        if (toCheck.value instanceof Error) {
```

Comment:
Though I don't think it necessarily a concern here, do we care that this will not identify errors that originate from an iframe?  Don't know how common of a use case that is.